Index: README.md
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Campus Compass Project\r\n\r\n#### 백석대학교 웹앱프로그래밍(화 4,5,6) 7조 팀프로젝트\r\n\r\n\r\n[Test Pages](http://https://ruan-p.github.io/buUniv-Campus_Compass/page.html)
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/README.md b/README.md
--- a/README.md	
+++ b/README.md	
@@ -3,4 +3,4 @@
 #### 백석대학교 웹앱프로그래밍(화 4,5,6) 7조 팀프로젝트
 
 
-[Test Pages](http://https://ruan-p.github.io/buUniv-Campus_Compass/page.html)
\ No newline at end of file
+[Click to Test page](http://https://ruan-p.github.io/buUniv-Campus_Compass/page.html)
\ No newline at end of file
