Index: localStorage.js
===================================================================
diff --git a/localStorage.js b/localStorage.js
deleted file mode 100644
--- a/localStorage.js	
+++ /dev/null	
@@ -1,11 +0,0 @@
-const $apibtn = document.querySelector("button #api_save");
-const apiData = document.getElementById("maps_api");
-
-// let fs = require("fs");
-
-const LOCAL_DATA = "";
-
-let onClickApi = () => {
-  console.log("click");
-  localStorage.setItem(LOCAL_DATA, apiData.value);
-};
